### ISSUE TITLE
option -w is used for both wait and width

### ISSUE THREAD

**Issue by s1291** [fathyb/html2svg#24]
Running: `docker run fathyb/html2svg --help`, one get the following:

```
Usage: html2svg [options] [command] <url>

Arguments:
  url                    URL to the web page to render

Options:
  -f, --full             capture the entire page
  -w, --wait <seconds>   set the amount of seconds to wait between the page
                         loaded event and taking the screenshot (default: 1)
  -w, --width <width>    set the viewport width in pixels (default: 1920)
  -h, --height <height>  set the viewport height in pixels (default: 1080)
  -f, --format <format>  set the output format, should one of these values:
                         svg, pdf, png, jpg, webp (default: "svg")
  --help                 display help for command

Commands:
  serve [options]

```
notice that `-w` is used for both `wait` and `width`. (This should be updated in `README.md` as well).

**Comment by s1291** [comment]
Thanks for the update. Please update the `README.md` file as well.

**Comment by fathyb** [comment]
The readme file has been updated and the updated flag has been published to the `1.0.0` release. Thanks for the report!
